maintainers/scripts/pluginupdate.py

```diff
@@ -328,12 +328,11 @@ def load_plugins_from_csv(
     return plugins
 
 
-
 def run_nix_expr(expr, nixpkgs: str, **args):
-    '''
+    """
     :param expr nix expression to fetch current plugins
     :param nixpkgs Path towards a nixpkgs checkout
-    '''
+    """
```

Comment:
aws this changed by black ?

Reply:
I ran `ruff format` which is [99.9%](https://docs.astral.sh/ruff/faq/#how-does-ruffs-formatter-compare-to-black) compatible with black (just because it was installed on my system), so whatever?